### ISSUE TITLE
aggregation fails when `points_per_worker` is not set

### ISSUE THREAD

**Issue by nickwg03** [NREL/reV#274]
**Bug Description**
When points_per_worker is not set in the JSON config, the aggregation stage fails with:
```
Usage: cli_sc_aggregation.py direct [OPTIONS] COMMAND [ARGS]...
Try 'cli_sc_aggregation.py direct --help' for help.

Error: Invalid value for '--points_per_worker' / '-ppw': None is not a valid integer
```

**Expected behavior**
A default value should be passed in so that the code doesn't fail if `points_per_worker is not set.

**Charge code**
WFED.11785.01.01.01

**Comment by grantbuster** [comment]
already fixed in the latest commit to master: https://github.com/NREL/reV/commit/a6ea5869de98856207722f06de59f2876a0edcfb

Was going to wait to put this in our next release.

**Comment by MRossol** [comment]
@nickwg03  FYI, about to release this fix in v0.4.51
